.gitignore

```diff
@@ -103,3 +103,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+test_nominal.ipynb
```

Comment:
Is there a reason for this one file being explicitly ignored?

Reply:
no i think that snuck through from someone having a local file during development they didn't want to commit.  Will tidy up

Reply:
[32e70cc](https://github.com/lvgig/tubular/pull/78/commits/32e70cce95cf23bb86d247baa44bbc0ad245feb6)